install/kubernetes/templates/istio-config.yaml.tmpl

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
```

Comment:
what does the moving from pilot to this new file do ? (it is probably cleaner if that config isn't just about pilot - is it - but it's not necessary for 0.2)

Reply:
Yes, it's much cleaner.